sched/file_upload_handler.cpp

```diff
@@ -289,16 +307,17 @@ int handle_file_upload(FILE* in, R_RSA_PUBLIC_KEY& key) {
     double max_nbytes=-1;
     char xml_signature[1024];
     int retval;
-    double offset=0, nbytes = -1, size;
+    double offset=0, nbytes = -1;
     bool is_valid, btemp;
 
     strcpy(name, "");
     strcpy(xml_signature, "");
     bool found_data = false;
     while (fgets(buf, 256, in)) {
-#if 1
-        log_messages.printf(MSG_NORMAL, "got:%s\n", buf);
-#endif
+        // intentionally set higher than debug as it may produce lots of output
+        if (config.fuh_debug_level > MSG_DEBUG) {
+            log_messages.printf(MSG_DEBUG, "got:%s\n", buf);
+        }
```

Comment:
Doesn't this ensure that this line never gets output?  Shouldn't this be >= MSG_DEBUG?

Reply:
No, the debug_levels in BOINC are defined as `enum { MSG_CRITICAL=1, MSG_WARNING, MSG_NORMAL, MSG_DEBUG, MSG_DETAIL };` which means the higher the level the more detailed output you get. In this case I should probably use `>= MSG_DETAIL` above (this level was added 3 moths ago and my code is older than that).

Reply:
I edited my answer above since Kevin added more info to his question and my original answer looked out of context.

Reply:
@ChristianBeer - sorry for the editing - thought I finished before you saw it.